hw/lint/data/common_lint_cfg.hjson

```diff
@@ -24,4 +24,9 @@
   report_cmd: "{proj_root}/hw/lint/tools/{tool}/parse-lint-report.py "
   report_opts: ["--repdir={build_dir}/lint-{tool} --outdir={build_dir}"]
   tool_srcs: []
+
```

Comment:
Yeah I know... it depends a bit whether you want to pull these highly tool-dependent parsing scripts into dvsim or not. Also, it is a bit easier to test if this can be called as a standalone script. We should maybe think about whether it made sense to specify something like a parser plugin at this point which has a defined interface - but the implementation can be custom.

Reply:
added a point for that here:
https://github.com/lowRISC/opentitan/issues/1727

Reply:
I actually meant the below keys that need to be needlessly set.

Reply:
ahh sorry then misunderstood you :). added another point for that to the issue mentioned above.